VirtualH89/Src/CPNetDevice.cpp

```diff
@@ -95,57 +95,69 @@ CPNetDevice::install_CPNetDevice(PropertyUtil::PropertyMapT& props)
     int         cid = 0xfe; // OK default if we have no network connections
 
     s = props["cpnetdevice_port"];
-
     if (s.empty())
     {
         return NULL;
     }
-
-    int port = strtoul(s.c_str(), NULL, 0);
-
+    int           port = 0x18;
+    unsigned long p    = strtoul(s.c_str(), NULL, 0);
+    if (p >= 0 && p <= 255)
+    {
+        port = p & 0x0ff;
```

Comment:
The mask tells the compiler that the conversion to a smaller int is intentional (suppresses warning). The range check insures that the user did not enter an invalid value.